packages/lit-node-client-nodejs/src/lib/lit-node-client-nodejs.ts

```diff
@@ -180,115 +174,57 @@ export class LitNodeClientNodeJs
   createCapacityDelegationAuthSig = async (
     params: CapacityCreditsReq
   ): Promise<CapacityCreditsRes> => {
-    const {
-      dAppOwnerWallet,
-      capacityTokenId,
-      delegateeAddresses,
-      uses,
-      domain,
-      expiration,
-      statement,
-    } = params;
+    // Useful log for debugging
+    if (!params.delegateeAddresses || params.delegateeAddresses.length === 0) {
```

Comment:
Not sure this log is accurate -- I think that `delegateeAddresses` being provided as an _empty array_ actually means that **_nobody** can use it_, but _no delegateeAddresses at all_ means that _anyone can use it_, right?

Reply:
Yea, I think you are technically correct if you pass an empty array to the rust function directly, I need to double check how it actually looks like in this siwe message after using the recap lib to add this auth sig in.

Reply:
> no delegateeAddresses at all means that anyone can use it, right

Yes that's the idea. Might be worth to have a test for it as many users wanted this feature